ingestion/src/main/java/feast/store/serving/redis/FeatureRowToRedisMutationDoFn.java

```diff
@@ -42,17 +45,26 @@ public FeatureRowToRedisMutationDoFn(Map<String, FeatureSetSpec> featureSetSpecs
 
   private RedisKey getKey(FeatureRow featureRow) {
     FeatureSetSpec featureSetSpec = featureSetSpecs.get(featureRow.getFeatureSet());
-    Set<String> entityNames =
+    List<String> entityNames =
         featureSetSpec.getEntitiesList().stream()
             .map(EntitySpec::getName)
-            .collect(Collectors.toSet());
+            .collect(Collectors.toList());
```

Comment:
Should we order this alphabetically?

Reply:
I'd like to follow whatever's on the spec.

Reply:
Ok nvm I've made it sorted.